src/snaskii.sh

```diff
@@ -19,17 +19,28 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-PROGRAM="./snaskii.bin"
+PROGRAM="snaskii.bin"
+HELP_SNASKII=""
+SKIP_INTRO=""
+SCENE_PATH="-d ../scenes"
 
 cd $(dirname $0)
 
+PARSED_ARGUMENTS=$(getopt -n Error -o hsd: -- "$@")
+
+eval set -- "$PARSED_ARGUMENTS"
+while :
+do
+  case "$1" in
+    -h)   HELP_SNASKII="-h"      ; shift   ;;
```

Comment:
Wrong aligment

Reply:
ammended the commit message and now correctly aligned